annotation-file-utilities/src/annotator/find/InheritedSymbolFinder.java

```diff
@@ -0,0 +1,211 @@
+package annotator.find;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.ElementFilter;
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+
+import annotations.util.JVMNames;
+import annotator.Source;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.LinkedHashMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Multimaps;
+
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.model.JavacElements;
+
+/**
+ * Provides static access to JVML descriptors of methods for which
+ *  implementations are inherited in a given class.
+ *
+ * @author dbro
+ */
+public class InheritedSymbolFinder {
+  private static Source source = null;
+
+  // class (flat) name -> method index
```

Comment:
Make these Javadoc comments.  (There is a Javadoc mode that also documents private members, and IDEs can also use the information.)

Also, what is a "flat name"?  That's not standard terminology and I feel it would be better to use terms with well-known meanings.

Finally, this comment has the wrong type.  The map is String -> String -> Element, but your documentation doesn't have the right number of ->.